assets/stylesheets/kitten/components/form/_text-area-wrapper.scss

```diff
@@ -15,6 +15,12 @@
     resize: vertical;
   }
 
+  .is-disabled {
+    .k-TextAreaWrapper__gradient {
+      display: none;
+    }
+  }
```

Comment:
Est-ce qu'on ne mettrait pas ce sélecteur à l'intérieur de `.k-TextAreaWrapper__gradient` pour éviter la répétition ? Genre `.is-disabled & { display: none; }` ?